src/components/commons/SkipToContent.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const SkipToContent = () => {
+  const { t } = useTranslation();
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
+
+  const linkStyle: any = {
+    transition: "opacity 0.2s ease-out",
+    opacity: isFocused ? "1" : "0",
```

Comment:
With opacity the button is clickable with the mouse by hovering over it, even if it is not visible.